framework/encode/custom_ags_wrappers.cpp

```diff
@@ -52,15 +52,25 @@ AMD_AGS_API AGSReturnCode agsInitialize(int                     agsVersion,
             shared_api_call_lock = D3D12CaptureManager::AcquireSharedApiCallLock();
         }
 
+        if (config != nullptr && (config->allocCallback != nullptr || config->freeCallback != nullptr))
```

Comment:
This changes application runtime behavior.  I recommend this be implemented instead in replay, by dropping any callback pointers passed into those functions there.

Reply:
Hi Brad, we actually tested both solutions with a real title, dropping the two callback pointers in capture or in replay. Both solutions worked. We chose to fix it in capture because we wanted to make the capture and the replay behaviors consistent. If you still think implementing this fix in replay is better, please let me know. Thanks.

Reply:
Thanks for the explanation. Now it is fixed at the replay side.